lib/app.jsx

```diff
@@ -507,6 +508,18 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 					</ContextMenu>
 				}
 				{ isAuthorized ?
+					<div className="simplenote-app">
+						<TopBar
+							note={selectedNote}
+							onNewNote={ this.onNewNote }
+							onTrashNote={this.onTrashNote}
+							onRestoreNote={this.onRestoreNote}
+							onShareNote={this.onShareNote}
+							onDeleteNoteForever={this.onDeleteNoteForever}
+							onRevisions={this.onRevisions}
+							onCloseNote={ () => this.props.actions.closeNote() }
+							onNoteInfo={ () => this.props.actions.toggleNoteInfo() }
```

Comment:
I'm definitely not happy of passing all those props there.
I've just copied them in order to get the top bar working without pushing a 1K LOC commit. :)